package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pretty-fast",
-  "version": "0.2.4",
+  "version": "0.2.5",
```

Comment:
Can we mention the version bump in the commit message?

Reply:
I'll split this into its own commit